### ISSUE TITLE
Coding presentation

### ISSUE THREAD

**Issue by Yilun0221** [Computational-Content-Analysis-2020/frequently-asked-questions-spring#27]
Just for confirmation!

I am to present my code next week (5.1). I need to present hw of week 3, right?

Much thanks!!!

**Comment by bhargavvader** [comment]
You need to present the HW for week 4 - this Friday (4/24/20), will be week 3 HW.

**Comment by Yilun0221** [comment]
> You need to present the HW for week 4 - this Friday (4/24/20), will be week 3 HW.

Is that 3-Classification.ipynb? 【Sorry I am so afraid that I got wrongly prepared....】

**Comment by bhargavvader** [comment]
No, that HW will be presented today - if you are presenting next week, you will be presenting 4-Semantic-Influence-Networks.ipynb.

**Comment by Yilun0221** [comment]
Got it! Thanks Bhargav!😊
